package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "preboot",
-  "version": "4.5.1",
-  "description": "Record and play back browser events",
-  "main": "__build/src/node/preboot_node.js",
-  "browser": "__dist/preboot_browser.js",
-  "typings": "preboot.d.ts",
+  "version": "5.0.0",
```

Comment:
This is a major version update. Everything is changed. Starting with 5.0.0 preboot only compatible with Angular. We can fork a diff version of preboot if there is interest in the community for the non-Angular version.